hrmcalcs2oo.py

```diff
@@ -0,0 +1,54 @@
+class hrmcalcs:
+    def __init__(self, timebeat, peakvalues, start_min, end_min):
+        """
+        :param timebeat: amount of time between consecutive heart beats.
+        Calculated by taking the difference of consecutive data points
+        in the peak values vector
+
+        :param peakvalues: array containing time points at which voltage
+        condition was met to detect a heart beat. All vals are in terms of
+        time(s)
+
+        :param start_min: User-inputted value for the "starting minute" of
+        average heart rate calculations, in minutes.
+
+        :param end_min: User-inputted value for the "ending minute" of average
+        heart rate calculations, in minutes.
+        """
+        self.timebeat = timebeat
+        self.start_min = start_min
+        self.end_min = end_min
+        self.peakvalues = peakvalues
+        self.instant_hr = []
+        self.average_hr = None
+        self.hrminstant()
+        self.hrmaverage()
+
+    def hrminstant(self):
+        """
+        This method calculates the instant heart rate values from timebeat,
+        the time occurrence of each heartbeat.
+
+        :return: This method returns a stored vector of instantaneous heart
+        rate values.
+        """
+        import numpy as np
+        for i in self.timebeat:
+            a = round((60/i), 1)
+            self.instant_hr.append(a)
```

Comment:
What happens if someone (the user of your class) calls `hrminstant` twice?
```python
h = hrmcalcs(...)
hr = h.hrminstant()
hr2 = h.hrminstant() # this could be called somewhere else by the user of your class after passing the h object around
print(h.instant_hr)
```